### ISSUE TITLE
adjust to texml changes

### ISSUE THREAD

**Issue by pkra** [AmerMathSoc/texml-to-html#382]
https://github.com/AmerMathSoc/texml/pull/146 brings a bunch of changes. Let's check what we need to do.

* [x] adjust to `<issn>` now   outside of `<journal-title-group>` => 2ebf40b 
  * journal-meta.js: extend selector (`querySelector('journal-meta>issn, journal-title-group>issn')`)
* [x] adjust to `<history>` now after `<page-range>` => no adjustment needed
* [x] Wrap `\tag` in `<target>`  => bc69a59
  * add closest('text-math') but breaks nested inline-math
  * 1 difficulty because of implicit text mode (target => tag => inline-formula)
* [x] Wrap tables in `table-wrap` instead of `<fig>`. => 7bdc46c
  * transformer: remove from pass-through & alias to fig
  * Note: potential data-ams-doc clash with div created by table (but downstream should be fine)
* [x] Replace `<ams-meta-group>` by `<kwd-group>` and `<compound-kwd>`? => 5058da4
  * [sample](https://github.com/AmerMathSoc/texml/pull/146#issuecomment-1244382566)
  * adds compound-kwd.js (to replace mscKey.js)
  * basically copies msc.js logic into kwd-group.js (using vocab to differentiate)
  * touches transformer, article-meta, 
* [x] Use `<private-char>` => 2484923 
  * for PUA-like SVGs, "true" inline-graphic, mostly math mode
  * pkra: maybe pass-through will be enough
* [x] `<title-group>` inside `<sec>` and then use it when there is an `<alt-title>` => no adjustment needed
  * currently not planned upstream but would just work (though ignored in articles right now).

---

rejected again upstream

* adjust to <tex-math> being replaced by a (TBD) tag
  * transformer: add to pass-through
  * otherwise: extend selectors (e.g., closest('tex-math, mathjax-tex'))
